api/handlers/jobs.ts

```diff
@@ -235,10 +238,8 @@ export async function snootyBuildComplete(event: APIGatewayEvent): Promise<APIGa
     await client.connect();
     const db = client.db(c.get<string>('dbName'));
     const jobRepository = new JobRepository(db, c, consoleLogger);
-    await jobRepository.updateWithCompletionStatus(jobId, null, false);
-    // Placeholder preview URL until we iron out the Gatsby Cloud site URLs.
-    // This would probably involve fetching the URLs in the db on a per project basis
-    const previewUrl = 'https://www.mongodb.com/docs/';
+    const { payload } = await jobRepository.updateWithCompletionStatus(jobId, null, false);
+    const previewUrl = getPreviewUrl();
```

Comment:
Should we call this function with the `payload` object?

Reply:
Oh yes, I was testing out it being undefined. So sorry, thanks for catching!!